File: GameEngine_IOS/GameEngine/Shaders/SkyBox/TSkyBoxAttribute.swift
import Foundation

/**
* Enumeration of attributes to the skyBox shader
*/
public enum TSkyBoxAttribute : Int {
    case
    
    /**
    * Position where to location attribute is going to be bind in the shader
    * program
    */
    position = 0,
    
    /*
     * Number of attributes in the skybox shader
     */
    numberOfAttributes;
}

File: GameEngine_Android/app/src/main/java/com/dferreira/gameEngine/shaders/skyBox/TSkyBoxAttribute.java
package com.dferreira.gameEngine.shaders.skyBox;

import com.dferreira.commons.IEnum;

/**
 * Enumeration of attributes to the skyBox shader
 */
public enum TSkyBoxAttribute implements IEnum {
	/**
	 * Position where to location attribute is going to be bind in the shader
	 * program
	 */
	position;

	/**
	 * The value of the enumeration
	 */
	@Override
	public int getValue() {
		return this.ordinal();
	}
}
